ecosystem/sep-00xx.md

```diff
@@ -0,0 +1,131 @@
+
+## Authentication Endpoint
+
+A web service indicates that it supports account authorization via this protocol by specifying `WEB_AUTHZ_ENDPOINT` in their [`stellar.toml`](sep-0001.md) file. This is how a wallet knows where to find the authorization server. A web server is required to implement the following behavior for the web authorization endpoint:
+
+* [`POST <WEB_AUTH_ENDPOINT>`](#authorize): exchange a SEP-10 key JWT for an account JWT
+
+## Cross-Origin Headers
+
+Valid CORS headers are necessary to allow web clients from other sites to use the endpoints. The following HTTP header must be set for all authentication endpoints, including error responses.
+
+```
+Access-Control-Allow-Origin: *
+```
+
+In order for browsers-based wallets to validate the CORS headers, as [specified by W3C]( https://www.w3.org/TR/cors/#preflight-request), the preflight request (OPTIONS request) must be implemented in all the endpoints that support Cross-Origin.
+
+### Authorize
+
+This endpoint accepts a Stellar account address and a SEP-10 JWT, validates it and responds with a augmented JWT containing additional claims about the level of control the signers in the SEP-10 JWT have over the Stellar account.
+
+Client submits as a HTTP POST request to `WEB_AUTHZ_ENDPOINT` using one of the following formats (both should be equally supported by the server):
+
+* Content-Type: `application/x-www-form-urlencoded`, body: `account=<Stellar account address>`)
+* Content-Type: `application/json`, body: `{"account": "<Stellar account address>"}`
+
+With the SEP-10 JWT in the `Authorization` header in the format `BEARER <JWT>`.
+
+To validate the request the server should complete the following steps. If any of the listed steps fail, then the authorization request must be rejected with HTTP Status Code 401. 
+* Verify the SEP-10 JWT is issued from a trusted issuer, is signed by that issuer with an expected signing method, and has not expired. The server should follow all JWT best practices to ensure the JWT can be trusted and that any other application specific claims are present and valid.
+* Verify the Stellar account is an active Stellar account.
+* Verify all the Stellar keys in the SEP-10 JWT `sub` field are signers of the Stellar account.
+
+Upon successful validation service responds with a session JWT, containing the following claims:
+
+* `iss` (the principal that issued a token, [RFC7519, Section 4.1.1](https://tools.ietf.org/html/rfc7519#section-4.1.1)) — a [Uniform Resource Identifier (URI)] for the issuer (`https://example.com` or `https://example.com/G...`)
+* `sub` (the principal that is the subject of the JWT, [RFC7519, Section 4.1.2](https://tools.ietf.org/html/rfc7519#section-4.1.2)) — the public keys of the authenticating client copied from the SEP-10 JWT
```

Comment:
Now that `sub` can contain more than 1 key,  we should clarify how they are separated inside the claim (comma vs space)

Reply:
That's clarified in #473.